docs/using-fit-on-fhir.md

```diff
@@ -2,18 +2,44 @@
 
 Congratulations. You have deployed Fit on FHIR to your Azure subscription and are ready to start collecting Google Fit Data. The first thing to do is have users on-board by authorizing sharing fitness data.
 
-## User authorization endpoint
+## User Authorization Endpoint
 
 When users navigate to the authorize endpoint, they are redirected to a Google sign-in page where they can enter their credentials and opt-in to sharing data with your Fit on FHIR instance. After they complete the authorization process, Fit on FHIR will continuously import the fitness data they opted to share. By default, data imports occur once per hour.
 
-**Your users can authorize sharing Google Fit data by navigating to [*https://authorize-{YOUR_BASE_NAME}.azurewebsites.net/api/googlefit/authorize*](#)**
-
 Your authorization endpoint is hosted by the Authorization Function. The endpoint to begin the authorization flow can be determined by viewing the Function overview page, finding the Function base URL and appending **api/googlefit/authorize** to the base.
 
+## Authorization Function Response Data
+
+To allow for users to be redirected to a Google sign-in page, the Authorization function returns a JSON object that contains two values:
+
+1. **AuthUrl** - This is the URL that should be used to navigate the user to the sign-in page.
+1. **ExpiresAt** - This is a timestamp that indicates when authorized access will expire, for the user that is trying to complete the authorization process.  Attempting to complete the process after this time will result in a 401 unauthorized response.
```

Comment:
"Attempting to complete the process after this time will result in a 401 unauthorized response" - I was thinking about this, and it might make more sense for the error to be returned as a parameter to the redirect_url.

From an end user (patient) perspective, they are using a website and they start the authorization process. Iif a 401 occurs, their web browser is navigated to a page that is basically broken. I think the right behavior, regardless of the outcome, would be after I complete the Google login process, that I would be navigated back to the website where I started. If we pass the error message back as a query parameter, the website implementor has the ability to show the. error message to the user or handle the issue and give the user further instruction.